README.adoc

```diff
@@ -306,14 +304,12 @@ include::finish/src/main/resources/META-INF/microprofile-config.properties[tags=
 ----
 
 ```
-{
-    "config_ordinal": 500,
-    "io.openliberty.guides.microprofile.inventory.inMaintenance": false,
-    "io.openliberty.guides.microprofile.system.inMaintenance": false,
-    "io.openliberty.guides.microprofile.inventory.testCustomConfigProperty": "developer@OpenLiberty.io",
-    "io.openliberty.guides.microprofile.inventory.testConfigOverwrite": "CustomSource"
+{   "config_ordinal": 500,
+    "io_openliberty_guides_inventory_inMaintenance": false,
+    "io_openliberty_guides_system_inMaintenance": false,
+    "io_openliberty_guides_email": "developer@OpenLiberty.io",
+    "io_openliberty_guides_testConfigOverwrite": "CustomSource"
```

Comment:
May be a better fake email is something like "admin@guides.openliberty.io" or something like that.  =)

Reply:
changed